specification/securityinsights/resource-manager/Microsoft.SecurityInsights/stable/2021-10-01/common/IncidentTypes.json

```diff
@@ -0,0 +1,66 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2021-10-01",
+    "title": "Common Incident types"
+  },
+  "paths": {},
+  "definitions": {
+    "IncidentSeverityEnum": {
+      "description": "The severity of the incident",
+      "enum": [
```

Comment:
>enum [](http://example.com/codeflow?start=7&length=4)

You removed the Critical enum value from this definition in previous API version. This is a breaking change and requires signoff from breaking change board. I've added the BreakingChangeReviewRequired label to your PR. Look in the comments for instructions on how to get the review.

Reply:
Thank you for the review @mentat9.
`Critical` isn't a real Incident severity and having it in the enum was a mistake already corrected and approved in the 2020-01-01 version ([https://github.com/Azure/azure-rest-api-specs/pull/15761/files](https://github.com/Azure/azure-rest-api-specs/pull/15761/files)). This branch was generated with the OpenAPI hub before the correction so it includes the mistake.

I applied a request to the breaking change board to review this PR.